### PR TITLE
Admin:  added `added_via_api` flag to `/clusters` endpoint

### PR DESCRIPTION
**Pull request by jmphilli**
Signed-off-by: jmphilli <jphillips@lyft.com>

### CODE REVIEW

**Comment by htuch** [comment]
How come there is no test for `/clusters`?

**Comment by jmphilli** [comment]
ya,... i noticed that as well..... there appears to be no existing test.

**Comment by mattklein123** [comment]
There is a "test" here: https://github.com/envoyproxy/envoy/blob/master/test/integration/integration_admin_test.cc#L110

Bonus points for actually parsing the results of that and doing something useful. :)

**Comment by mattklein123** [comment]
P.S. you could also add a better test to admin_impl_test directly. Either way.

File: docs/operations/admin.rst [hunk 1]
```diff
@@ -30,6 +30,8 @@ modify different aspects of the server.
       are presented. Both of these values could be ``-1`` if there was not enough data to calculate them in the last
       :ref:`interval<config_cluster_manager_cluster_outlier_detection_interval_ms>`.
 
+    - ``added_via_api`` flag - ``false`` if the cluster was added via static configuration, ``true`` if it was added via the :ref:`CDS<config_cluster_manager_cds>` api.
```
Comment (mattklein123) [inline docs/operations/admin.rst]: nit: please line break this at ~100 cols.

**Pull request merged**